.github/workflows/pospell.yml

```
@@ -0,0 +1,42 @@
+name: pospell
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  fix-spell:
+    name: Fix spelling
+    runs-on: ubuntu-latest
+    if: ${{ github.event.comment.body == 'pospell-fix'}}
```

```suggestion
    runs-on: ubuntu-latest
    if: ${{ github.event.issue.pull_request }}
    if: ${{ github.event.comment.body == 'pospell-fix'}}
```
Para limitar a las PR.